packages/vm/src/runTx.ts

```diff
@@ -301,7 +301,7 @@ async function _runTx(this: VM, opts: RunTxOpts): Promise<RunTxResult> {
   const { nonce, balance } = fromAccount
 
   // EIP-3607: Reject transactions from senders with deployed code
-  if (fromAccount.codeHash !== KECCAK256_NULL) {
+  if (!fromAccount.codeHash.equals(KECCAK256_NULL)) {
```

Comment:
Ah, have done a commit-by-commit review, just seeing the update, so forget about the first comment. 😄